lib/resources/servers/actions/index.js

```diff
@@ -44,6 +44,22 @@ class Actions {
     return;
   }
 
+  lockDevice(serverId) {
+    const headers = this.LatitudeSh._headers;
+    return this.LatitudeSh._post(
+      '/servers/' + serverId + '/lock',
+      headers
+    );
+  }
+
+  unlockDevice(serverId) {
```

Comment:
We can use only the action name, like the other actions `reinstall`, `scheduleDeletion`. Could be only `lock` and `unlock`

Reply:
Done